crates/phactory/src/system/mod.rs

```diff
@@ -1544,6 +1566,30 @@ impl<P: pal::Platform> System<P> {
         self.check_retirement();
         Ok(())
     }
+
+    pub(crate) fn apply_side_effects(
```

Comment:
Where is it used?

Reply:
It is called here:
https://github.com/Phala-Network/phala-blockchain/pull/915/files#diff-82cdfe9b271153bf6b4c581e0f64a6c6b8fc86fe0298b00c2081a0b7287b6bfaR486
Then here:
https://github.com/Phala-Network/phala-blockchain/pull/915/files#diff-82cdfe9b271153bf6b4c581e0f64a6c6b8fc86fe0298b00c2081a0b7287b6bfaR966